src/SqlBindingUtilities.cs

```diff
@@ -54,6 +54,15 @@ public static string GetConnectionString(string connectionStringSetting, IConfig
             return connectionString;
         }
 
+        public static string GetWebSiteName(IConfiguration configuration)
+        {
+            if (configuration == null)
```

Comment:
What happens for local development? Is website site name set?

Reply:
WEBSITE_SITE_NAME value is null for local development.